xsuite/src/world/fsworld.test.ts

```diff
@@ -223,6 +223,30 @@ test.concurrent("FSWorld.getAccountBalance", async () => {
   expect(await world.getAccountBalance(wallet)).toEqual(1234n);
 });
 
+test.concurrent("FSWorld.getAccountValue - Invalid key", async () => {
+  using world = await FSWorld.start();
+  const wallet = await world.createWallet({ kvs: [[e.Str("n"), e.U(1)]] });
+  await expect(world.getAccountValue(wallet, "n")).rejects.toThrow(
+    "invalid byte",
+  );
+});
+
+test.concurrent("FSWorld.getAccountValue - Non-existent key", async () => {
+  using world = await FSWorld.start();
+  const wallet = await world.createWallet();
+  await expect(
+    world.getAccountValue(wallet, e.Str("n").toTopHex()),
+  ).rejects.toThrow("fetching value error: trie is nil");
+});
+
+test.concurrent("FSWorld.getAccountValue", async () => {
+  using world = await FSWorld.start();
+  const wallet = await world.createWallet({ kvs: [[e.Str("n"), e.U(1)]] });
```

Comment:
For the value you can directly put some hex (e.g. "11") as there is no need for encodable